docs/installation/howto/optional.rst

```diff
@@ -15,66 +15,109 @@ For a deeper explanation of what optional dependencies are and how they are decl
 
 The following optional dependencies are supported:
 
-* **dev** - packages that aid in the development of Twisted itself.
+tls
+   Packages that are needed to work with TLS:
 
-  * `pyflakes`_
-  * `twisted-dev-tools`_
-  * `python-subunit`_
-  * `Sphinx`_
-  * `TwistedChecker`_, only available on python2
```

Comment:
Not the responsibility of this change, but, I think that we should *not* actually re-enumerate all dependencies in this document any more. Just list the extras, and let the package metadata speak for itself if people want to know specifics. We can't necessarily keep this up to the minute up-to-date if someone adds a transitive dependency, for example.